changelogs/fragments/2086_Programs_fails_when_need_access_to_the_first_and_second_columns.yml

```diff
@@ -0,0 +1,4 @@
+minor_changes:
+  - zos_mvs_raw  - Add reserved_cols option and validate that the module get access to modify dd_content option
+    base on the value, if not retain the previous behavior or work.
```

Comment:
Suggest, if you have time, use 'before': "Before this addition, you could not put anything in columns 1 or 2."  We located a customer who needed to put information on Column 1.  Reserved_cols now allows you co control how many (if any) columns are reserved, giving the user more control over their dd statements.